agent/roles/manifests/tasks/install-config.yml

```diff
@@ -15,6 +15,12 @@
     dest: "{{ install_path }}/install-config.yaml"
   when: platform_type == "baremetal"
 
+- name: write the install-config.yaml
+  template:
+    src: "templates/install-config_vsphere_yaml.j2"
+    dest: "{{ install_path }}/install-config.yaml"
+  when: platform_type == "vsphere"
```

Comment:
I think you also need to add an `or platform_type != "vsphere"` to line 10 above

Reply:
Thanks!